internal/pkg/deploy/cloudformation/stack/lb_web_svc.go

```diff
@@ -43,50 +44,53 @@ type LoadBalancedWebService struct {
 	manifest     *manifest.LoadBalancedWebService
 	httpsEnabled bool
 
+	// Fields for LoadBalancedWebService that needs a Network Load Balancer.
+	dnsDelegationEnabled   bool
```

Comment:
Maybe one thing to clarify as field comments is the difference between `httpsEnabled` and `dnsDelegationEnabled`? 🥺